src/components/InformationBoxComponent/InformationBoxComponent.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./InformationBoxComponentStyles";
+
+const InformationBox = ({ title, imagePath, description }) => {
+  return (
+    <div style={styles.container}>
```

Comment:
here it should be `className={styles.container}` only and not `style={styles.container}>` 


The way of using .module.css and .css is just same, in css its just `.classname` and when we import styles from .module.css it is `styles.classname`